ci/federalist-pipeline.yml

```diff
@@ -0,0 +1,279 @@
+---
+############################
+#  SHARED
+
+env-cf: &env-cf
+  CF_API: https://api.fr.cloud.gov
+  CF_USERNAME: ((production-cf-username))
+  CF_PASSWORD: ((production-cf-password))
+  CF_ORG: gsa-18f-federalist
+  CF_SPACE: production
+
+node-image: &node-image
+  platform: linux
+  image_resource:
+    type: docker-image
+    source:
+      repository: node
+      tag: 16.14
+
+cf-image: &cf-image
+  platform: linux
+  image_resource:
+    type: docker-image
+    source:
+      repository: 18fgsa/concourse-task
+
+test: &test
+  - in_parallel:
+    - get: redis
+      params: {save: true}
+    - get: node
+      params: {save: true}
+  - task: prepare
+    config:
+      <<: *node-image
+      inputs: [name: src]
+      outputs: [name: src]
+      run:
+        dir: src
+        path: bash
+        args:
+          - -c
+          - |
+            yarn --pure-lockfile
+            yarn eslint
+  - task: test
+    privileged: true
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source:
+          repository: karlkfi/concourse-dcind
+      inputs:
+        - name: src
+        - name: redis
+        - name: node
+      run:
+        dir: src
+        path: ci/docker/entrypoint.sh
+        args:
+          - bash
+          - -ceux
+          - |
+            pushd ..
+              docker load -i redis/image
+              docker tag "$(cat redis/image-id)" "$(cat redis/repository):$(cat redis/tag)"
+              docker load -i node/image
+              docker tag "$(cat node/image-id)" "$(cat node/repository):$(cat node/tag)"
+            popd
+            docker-compose -f ci/docker/docker-compose.yml run app app/ci/tasks/test.sh
+            docker-compose -f ci/docker/docker-compose.yml down
+            docker volume rm $(docker volume ls -q)    
+
+
+############################
+#  JOBS
+
+jobs:
+
+  - name: set-pipeline
+    plan:
+    - get: src
+      resource: src-production
+      trigger: true
+    - set_pipeline: federalist-builder
+      file: src/ci/federalist-pipeline.yml
+
+  - name: test-pr-main
+    plan:
+    - get: src
+      resource: pr-main
+      trigger: true
+      version: every
+
+    - put: src
+      resource: pr-main
+      params:
+        path: src
+        status: pending
+        base_context: concourse
+        context: test
+
+    - do: *test
+
+    on_failure:
+      put: src
+      resource: pr-main
+      params:
+        path: src
+        status: failure
+        base_context: concourse
+        context: test
```

Comment:
We should update these `context` values to something like `test-federalist` so the pages production and the federalist production checks don't conflict with each other.